catimg.spec

```diff
@@ -0,0 +1,42 @@
+Name:    catimg
+Version: 2.0
+Release: 1%{?dist}
```

Comment:
Yeah. On my system (Fedora 22) this adds `.fc22` to the version like this:

`catimg-2.0-1.fc22`